Sources/SafeDICore/Visitors/InstantiableVisitor.swift

```diff
@@ -57,9 +57,8 @@ public final class InstantiableVisitor: SyntaxVisitor {
             let patterns = node
                 .bindings
                 .filter {
-                    $0.initializer == nil
-                    && $0.accessorBlock == nil
-                    && !$0.isOptionalAndUninitialized
```

Comment:
`isOptionalAndUninitialized` was named incorrectly. It was really checking whether something was optional _or_ uninitialized